src/components/sort-by.tsx

```diff
@@ -29,22 +35,26 @@ export const SortBy = () => {
 
       <div
         className={cn(
-          "absolute right-0 top-12 hidden w-[60vw] flex-col gap-3 rounded-md border border-techie-gray-300 bg-white p-4 py-5 shadow-md md:w-[150%]",
-          open && "flex",
+          'absolute right-0 top-12 hidden w-[60vw] flex-col gap-3 rounded-md border border-techie-gray-300 bg-white p-4 py-5 shadow-md md:w-[150%]',
+          open && 'flex',
         )}
       >
-        <button className="text-start" onClick={() => {}}>
+        <button className="text-start font-text" onClick={() => {}}>
           My favorites
         </button>
-        <button className="text-start" onClick={() => {}}>
-          Price - from high to low
-        </button>
-        <button className="text-start" onClick={() => {}}>
-          Price - from low to high
-        </button>
-        <button className="text-start" onClick={() => {}}>
-          Alphabetical order
-        </button>
+        {options.map((option, index) => (
+          <button
+            key={index}
```

Comment:
[Avoid using index as key](https://react.dev/learn/rendering-lists#why-does-react-need-keys)

Reply:
Some more documentation about this on our [standards](https://lightit.slite.com/app/docs/LBfbP_38cP-oT2?noteModalId=9qRM-JqhgCpj1q)!

Reply:
Thank you for the feedback! I’m aware that using the index as a key is not ideal due to potential issues with reordering. In this case, the options array is static and only contains 4 objects, so I used the index as a quick solution. However, I agree that it's better to follow best practices, so I'll update the code by assigning a unique id to each object and use that as the key. Thanks again for the reminder!

Reply:
you can use the option if its unique